src/resty/gcp/init.lua

```diff
@@ -86,13 +93,10 @@ local function build_request(accesstoken, apiDetail, baseUrl, params, requestBod
         path_template = apiDetail.path
     end
 
-    local path = baseUrl .. template_expansion(path_template, params)
+    local path, consumed_parameters = template_expansion(path_template, params)
+    path = baseUrl .. path
     local query
 
-    if apiDetail.flatPath then
```

Comment:
I must apologize for not reading the document before writing this...
https://developers.google.com/discovery/v1/reference/apis#:~:text=resources.(key).methods.(key).flatPath